pyjetty/alice_analysis/analysis/user/substructure/run_analysis.py

```diff
@@ -174,75 +180,79 @@ def create_output_subdir(self, output_dir, name):
 
         obs_setting = self.obs_settings[i]
         grooming_setting = self.grooming_settings[i]
         obs_label = self.utils.obs_label(obs_setting, grooming_setting)
-        reg_param_final = self.utils.get_reg_param(self.obs_settings, self.grooming_settings, self.obs_subconfig_list, self.obs_config_dict, obs_label, jetR)
 
         # Compute systematics and attach to main results
         if self.do_systematics:
-          self.compute_systematics(jetR, obs_label, obs_setting, grooming_setting, reg_param_final)
+            self.compute_systematics(jetR, obs_label, obs_setting, grooming_setting)
 
         # Plot result for each individial subconfiguration
         if self.do_plot_final_result:
           self.plot_single_result(jetR, obs_label, obs_setting, grooming_setting) # You must implement this
-          
+
       # Plot final results for all subconfigs
       if self.do_plot_final_result:
         self.plot_all_results(jetR) # You must implement this
-   
+
     # Plot additional performance plots
     if self.do_plot_performance:
       self.plot_performance() # You must implement this
-   
+
   #----------------------------------------------------------------------
   def perform_unfolding(self):
     print('Perform unfolding for all systematic variations: {} ...'.format(self.observable))
-    
+
     for systematic in self.systematics_list:
-    
+
       output_dir = getattr(self, 'output_dir_{}'.format(systematic))
       data = self.main_data
       response = self.main_response
-      
+
       main_response_location = os.path.join(getattr(self, 'output_dir_main'), 'response.root')
       rebin_response = self.check_rebin_response(output_dir)
-      
+
       prior_variation_parameter = 0.
       truncation = False
       binning = False
-    
+
       if systematic == 'trkeff':
         response = self.trkeff_response
-      if systematic == 'prior1':
+      elif systematic == 'prior1':
         prior_variation_parameter = self.prior1_variation_parameter
         shutil.copyfile(main_response_location, os.path.join(output_dir, 'response.root'))
         rebin_response = False
-      if systematic == 'prior2':
+      elif systematic == 'prior2':
         prior_variation_parameter = self.prior2_variation_parameter
         shutil.copyfile(main_response_location, os.path.join(output_dir, 'response.root'))
         rebin_response = False
-      if systematic == 'truncation':
+      elif systematic == 'truncation':
         truncation = True
-      if systematic == 'binning':
+      elif systematic == 'binning':
         binning = True
-      
-      analysis = roounfold_obs.Roounfold_Obs(self.observable, data, response, self.config_file, output_dir, self.file_format, rebin_response=rebin_response, prior_variation_parameter=prior_variation_parameter, truncation=truncation, binning=binning)
+
+      analysis = roounfold_obs.Roounfold_Obs(self.observable, data, response, self.config_file,
```

Comment:
More funny spacings here...be sure to check the diff on github before making the PR to check for these kind of things